ODIN_II/regression_test/benchmark/verilog/keywords/generate_endgenerate/generate_else_if.vh

```diff
@@ -0,0 +1,13 @@
+module simple_op(a,b,out);
+	input a,b;
+	output out;
+	
+generate 
+	if(`en)begin:logic
```

Comment:
tags don't work yet! :P this should be a syntax test (also we need to revamp the syntax bm).

Reply:
If I take out the label will it work? If so, should I do that. This one was inspired by an example in the verilog standard